nemo/collections/asr/models/ctc_models.py

```diff
@@ -35,6 +35,17 @@
 class EncDecCTCModel(ASRModel):
     """Encoder decoder CTC-based models."""
 
+    @classmethod
+    def list_available_models(cls) -> Optional[PretrainedModelInfo]:
```

Comment:
can we support `location` to be passed by the user as a local path? Or do we not want .nemo files to be downloaded manually at all?

Reply:
will add it in the next commit

Reply:
actually, I'm not exactly sure what you mean. Where will names and description come from?

Yes, the user can download .nemo file from NGC and to "model = ModelClass.restore_from("file_from_NGC.nemo")"

Reply:
Ah that second part is what I wanted. Very convenient. Nice !